interactions/api/gateway/client.py

```diff
@@ -290,10 +290,10 @@ def _dispatch_event(self, event: str, data: dict) -> None:  # sourcery no-metric
                             )
                             if _type:
                                 if isinstance(option, dict):
-                                    _type[option["value"]]._client = self._http
+                                    _type[option["value"]].client = self._http
```

Comment:
Shouldn't this (and pretty much the entire file here) be using `_client`?

Reply:
I believe `.client` is correct there, since the Context classes don't have it underscored

Reply:
I think it's for an option value here, not for context itself?

Reply:
I've tested with options, and they work
That line isn't even touched in this pr